subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_rai.c

```diff
@@ -122,8 +122,10 @@ int lwm2m_rai_no_data(void)
 		}
 
 		if (ctx->sock_fd >= 0) {
+			int dummy;
```

Comment:
```suggestion
			int dummy = 1;
```

Even when we don't use the value, initialize it, so at least we don't get warnings of using uninitialized variables, if static analyzers see this.